<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4071" started-at="2016-11-07T06:51:06Z" finished-at="2016-11-07T06:51:10Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4071" started-at="2016-11-07T06:51:06Z" finished-at="2016-11-07T06:51:10Z">
      <class name="com.test.freelance.rest.TestUsingHttpClient">
        <test-method status="PASS" signature="init()[pri:0, instance:com.test.freelance.rest.TestUsingHttpClient@55f3ddb1]" name="init" is-config="true" duration-ms="77" started-at="2016-11-07T12:21:06Z" finished-at="2016-11-07T12:21:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="FAIL" signature="testPost()[pri:0, instance:com.test.freelance.rest.TestUsingHttpClient@55f3ddb1]" name="testPost" duration-ms="3947" started-at="2016-11-07T12:21:06Z" finished-at="2016-11-07T12:21:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Verify the text in the twitter message]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Verify the text in the twitter message
	at org.junit.Assert.fail(Assert.java:93)
	at org.junit.Assert.assertTrue(Assert.java:43)
	at com.test.freelance.rest.TestUsingHttpClient.validateTitle(TestUsingHttpClient.java:119)
	at com.test.freelance.rest.TestUsingHttpClient.testPost(TestUsingHttpClient.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPost -->
      </class> <!-- com.test.freelance.rest.TestUsingHttpClient -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
